scripts/squeezed.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Manages host memory by ballooning VMs
+Description=XCP ballooning daemon
+After=xenstored.service message-switch.service syslog.target
+Wants=xenstored.service message-switch.service syslog.target
+PartOf=toolstack.target
+
+[Service]
+Environment="LD_PRELOAD=/usr/lib64/libjemalloc.so.2"
+Environment="MALLOC_CONF=background_thread:true,dirty_decay_ms:100,narenas:1,tcache:false"
+Environment=OCAMLRUNPARAM=b
+EnvironmentFile=-/etc/sysconfig/squeezed
+ExecStart=/usr/sbin/squeezed $SQUEEZED_OPTIONS
```

Comment:
Does this work? Where is `$SQUEEZED_OPTIONS` coming from?

Reply:
I have no idea, I just moved the file, seems like an existing bug

Reply:
`EnvironmentFile=-/etc/sysconfig/squeezed`
Looks this file define the Environment, and default to empty